client/sites-dashboard/components/wpcom-smp-commands.tsx

```diff
@@ -983,6 +997,43 @@ export const useCommandsArrayWpcom = ( {
 			},
 			icon: pluginsIcon,
 		},
+		{
+			name: 'changePlan',
+			label: __( 'Change plan on a site' ),
```

Comment:
```suggestion
			label: __( 'Change site plan' ),
```

I think 'Change plan on a site' is to close to a full sentence. The other commands are more focused on statements.

Reply:
Changed in https://github.com/Automattic/wp-calypso/pull/85083/commits/47e470467aa9010a7f2697c44147d98d69aef2d4